docs/content/en/docs/pipeline-stages/healthcheck.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Deploy Healthchecks"
+linkTitle: "Deploy Healthchecks"
+weight: 50
+featureId: deploy.status_check
+aliases: [/docs/how-tos/healthchecks]
+---
+
+This page describes how skaffold runs healthchecks for deployed resources - waits for them to stabilize, and reports errors on failure.
+
+### Overview
+
+Commands that trigger a deployment like `skaffold dev`, `skaffold deploy`, `skaffold run`, and `skaffold apply` perform a `healthcheck` for select Kubernetes resources and wait for them to be stable.
+
+Skaffold supports the following resource types for monitoring status after creation in the cluster:
+* [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/)
+* [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) 
+* [`Stateful Sets`](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) 
+* [`Google Cloud Config Connector resources`](https://cloud.google.com/config-connector/docs/overview)
+
+{{<alert title="Note">}}
+`healthcheck` is enabled by default; it can be disabled with the `--status-check=false`
+flag, or by setting the `statusCheck` field of the deployment config stanza in
+the `skaffold.yaml` to false.
+
+If there are multiple skaffold `modules` active, then setting `statusCheck` field of the deployment config stanza will only disable healthcheck for that config. However using the `--status-check=false` flag will disable it for all modules.
+{{</alert>}}
+
+To determine if a `Deployment` or `StatefulSet` resource is up and running, Skaffold relies on `kubectl rollout status` to obtain its status. For standalone `Pods` it checks that the pod and its containers are in a `Ready` state.
```

Comment:
This description is necessary but is buried here.  Should either move it into the list above or create a separate table.